server/src/test/java/org/cloudfoundry/identity/uaa/provider/oauth/ExternalOAuthAuthenticationManagerTest.java

```diff
@@ -158,8 +157,8 @@ public void getExternalAuthenticationDetails_whenProviderHasSigningKey_throwsWhe
         expectedException.expectMessage("Could not verify token signature.");
 
         Map<String, Object> header = map(
-                entry(ALG, "HS256"),
-                entry(KID, "oidc-provider-key")
+                entry(HeaderParameterNames.ALGORITHM, "HS256"),
+                entry(HeaderParameterNames.KEY_ID, "oidc-provider-key")
```

Comment:
It feels like the 2 values "HS256" and "oidc-provider-key" could also be made constants.

Reply:
done